package.json

```diff
@@ -65,6 +65,7 @@
     "@uppy/utils": "file:packages/@uppy/utils",
     "@uppy/webcam": "file:packages/@uppy/webcam",
     "@uppy/xhr-upload": "file:packages/@uppy/xhr-upload",
+    "adm-zip": "0.4.13",
```

Comment:
In the past we've put build dependencies in `devDependencies` in the root repo, and only put the uppy packages in `dependencies`. it doesn't make a difference functionally but for consistency's sake it would be best to move this to devDependencies too